http/node/fetch.js

```diff
@@ -67,14 +67,23 @@ function fetch(url, options = {}) {
 	let promise = new Promise((resolve, reject) => {
 		decipherUrl(url, options);
 
-		if(options && options.method && options.method.toLowerCase() !== "get"){
-			throw Error("http.fetch on nodejs environment should be used only for GET requests for the moment. Use http.doPost instead.");
-		}
-
 		let request = protocol.request(url, options, (response) => {
 			resolve(new Response(request, response));
 		});
 
+		if (options.body) {
```

Comment:
The request body is needed for the notification subsystem. The code is based on the `generateMethodForRequestWithData()`